luaui/Widgets/gui_camera_anchors.lua

```diff
@@ -8,15 +8,16 @@ function widget:GetInfo()
 	return {
 		name      = "Camera Anchors",
 		desc      = "Adds keybindings for Camera Anchors",
-		author    = "badosu",
-		date      = "Oct 06, 2021",
+		author    = "badosu, lonewolfdesign",
+		date      = "Mar 12, 2023",
```

Comment:
Not sure if the convention is date of creation, most drastic modification or last update but I really don't care to be honest.